### PR TITLE
timeslot component

### PR DESCRIPTION
**Pull request by Yarin18**
I didn't put the text styles in the object; since it doesn't exist on this version of develop yet, once that will be merged in I will move those constants.

### CODE REVIEW

File: composeApp/src/androidMain/kotlin/com/wisemen/chargehub/ui/components/Timeslot.kt [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.wisemen.chargehub.R
+import com.wisemen.chargehub.ui.theme.AppTheme
+import com.wisemen.chargehub.ui.theme.Colors
+
+@Preview(showBackground = true)
+@Composable
+fun TimeSlot(
```
Comment (DanneD-Wisemen) [inline composeApp/src/androidMain/kotlin/com/wisemen/chargehub/ui/components/Timeslot.kt:38]: This component is too large

File: composeApp/src/androidMain/kotlin/com/wisemen/chargehub/ui/components/Timeslot.kt [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+                    Modifier.padding(start = 12.dp, top = 10.dp),
+                ) {
+                    Row {
+                        Text(
+                            text = stringResource(R.string.laadpaal, chargeHubName),
+                            style = cardTitleStyle
+                        )
+                        Spacer(modifier = Modifier.weight(1F))
+                        Icon(
+                            Icons.AutoMirrored.Filled.ArrowForwardIos,
```
Comment (DanneD-Wisemen) [inline composeApp/src/androidMain/kotlin/com/wisemen/chargehub/ui/components/Timeslot.kt]: iOS icon in Android? :)

File: composeApp/src/androidMain/kotlin/com/wisemen/chargehub/ui/components/Timeslot.kt [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+
+private val textTextStyle = TextStyle(
```
Comment (DanneD-Wisemen) [inline composeApp/src/androidMain/kotlin/com/wisemen/chargehub/ui/components/Timeslot.kt:159]: Why are these added here? Weren't these already created? These should not belong here

File: composeApp/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="jouw_tijdslot">JOUW TIJDSLOT</string>
```
Comment (DanneD-Wisemen) [inline composeApp/src/main/res/values/strings.xml]: Make the keys in English

**Review by DanneD-Wisemen — CHANGES_REQUESTED** [review]

File: composeApp/src/androidMain/kotlin/com/wisemen/chargehub/ui/components/Timeslot.kt (hunk 3 above)
Comment (Yarin18) [reply composeApp/src/androidMain/kotlin/com/wisemen/chargehub/ui/components/Timeslot.kt:159]: I added them there because that object isn't on develop yet, once that is in (Textfields PR), I will move add them there

File: composeApp/src/androidMain/kotlin/com/wisemen/chargehub/ui/components/Timeslot.kt (hunk 1 above)
Comment (Yarin18) [reply composeApp/src/androidMain/kotlin/com/wisemen/chargehub/ui/components/Timeslot.kt:38]: Divided it into multiple smaller components, is this better?

File: composeApp/src/androidMain/kotlin/com/wisemen/chargehub/ui/components/Timeslot.kt (hunk 3 above)
Comment (DanneD-Wisemen) [reply composeApp/src/androidMain/kotlin/com/wisemen/chargehub/ui/components/Timeslot.kt:159]: Maybe add a TODO for this

**Review by DanneD-Wisemen — APPROVED** [review]
